coordenadas.js

```diff
@@ -0,0 +1 @@
+var data='[{"name": "Doncella","lat": 19.46505,"lng": -70.687903,"info": "Aqui esta Doncella"},{"name": "Intellisys","lat": 19.47023552,"lng": -70.67755723,"info": "Aqui esta Intellisys"},{"name": "Hache","lat": 19.46914853,"lng": -70.6868484,"info": "Aqui esta plaza Hache"}]';
```

Comment:
This needs to be formatted. Also add a line at the end of the file.
```suggestion
var data='[
  {
    "name": "Doncella", 
    "lat": 19.46505,
    "lng": -70.687903,
    "info": "Aqui esta Doncella"
  },
  {
    "name": "Intellisys",
    "lat": 19.47023552,
    "lng": -70.67755723,
    "info": "Aqui esta Intellisys"},
  {
    "name": "Hache",
    "lat": 19.46914853,
    "lng": -70.6868484,
    "info": "Aqui esta plaza Hache"
  }
]';
```